docs/source/host_challenge.md

```diff
@@ -162,6 +162,97 @@ Finally run the `./run.sh` script in the bundle. It will generate a `challenge_c
 
 **Congratulations!** you have submitted your challenge configuration for review and [EvalAI team](https://eval.ai/team) has notified about this. [EvalAI team](https://eval.ai/team) will review and will approve the challenge.
 
+## Host a challenge with remote evalution
+
+The steps related to challenge creation will be similar to the steps mentioned above for different types of challenges. This section specifically deals with the bits and pieces required to set up remote challenge evaluation on your own instances.
+
+### Step 1: Modify challenge configuration
+
+Set the `remote_evaluation` parameter to `True` in [`challenge_config.yaml`](https://github.com/Cloud-CV/EvalAI-Starters/blob/621f0cb37b2f1951613c9b6c967ce35be55d34c8/challenge_config.yaml#L12). The `challenge_config.yaml` file defines all the different settings of a challenge and you can edit other things based on your requirement.
+
+Make sure that following fields are set correctly:
+
+- `remote_evaluation` is set to `True`
+
+This will ensure that the challenge worker is aware that the evaluation is to be performed remotely.
+
+### Step 2: Fetch details for the challenge
+
+Now, go to [EvalAI](https://eval.ai) to fetch the following details -
+
+1. `auth_token` - Go to [profile page](https://eval.ai/web/profile) after logging in and click on `Get your Auth Token` to copy your auth token.
+2. `evalai_api_server` - Use `https://eval.ai` for production server and `https://staging.eval.ai` for staging server
+
+<img src="_static/img/github_based_setup/evalai_profile.png"><br />
+
+Once the challenge is up on the server, you can go to Manage Tab and fetch the following:
+
+3. `queue_name`: The queue name for the worker which will be used to receive submissions from participants.
+4. `challenge_pk`: The primary key for the challenge.
+
+<img src="_static/img/remote_evlaluation_meta.png"><br />
+
+### Step 3: Edit remote evaluation script
+
+Next step is to edit the challenge evaluation script that decides what metrics the submissions are going to be evaluated on for different phases. The evaluation script template is different for the remote evaluation and can be found here: [Remote Challenge Evaluation Script](https://github.com/Cloud-CV/EvalAI-Starters/blob/master/remote_challenge_evaluation/evaluation_script_starter.py).
```

Comment:
Refer [this documentation](https://evalai.readthedocs.io/en/latest/evaluation_scripts.html) to see how to write the setting up evaluation script part.

Reply:
Should "Writing a Remote Evaluation Script" be along with the evaluation script documentation?  It makes more sense I guess. We can add some basic info here and link to that documentation.